### ISSUE TITLE
Simple example with Stack + non base package?

### ISSUE THREAD

**Issue by fabricatedmath** [mvdan/hint#55]
Was wondering if anybody had an example with paths to use a package like vector with hint + Stack. I used ```unsafeRunInterpreterWithArgs``` with package-db arg, suggested in #44, but hint still can't find the library, not sure if this has something to do with differences with raw cabal vs Stack.

Thanks

**Comment by gelisam** [comment]
I had to use `unsafeRunInterpreterWithArgs` at the time, but I just tried and when compiling with stack, the regular `runInterpreter` works just fine:

    import Language.Haskell.Interpreter

    main :: IO ()
    main = do
      r <- runInterpreter $ do
        setImportsQ [("Prelude", Nothing), ("Data.Vector", Just "V")]
        eval "V.iterateN 10 (+1) 0 V.! 5"
      print r

I didn't have to put something special in `stack.yaml` nor `package.yaml`; I didn't even have to list `vector` as a dependency, but that's probably because it's a dependency of another package I built using the same lts, so the compiled files are already in the package-db.

The one thing I did have to do in order for this to work was to run my executable with `stack exec -- myexe`, not as `stack install && myexe`. Doing so sets up an environment variable which tells hint (or rather the underlying ghc library) where to find the relevant package databases.
